Sync/Synchronizers/ClientsSynchronizer.swift

```diff
@@ -346,3 +345,31 @@ public class ClientsSynchronizer: TimestampedSingleCollectionSynchronizer, Synch
             >>> { deferMaybe(.Completed) }
     }
 }
+
+private extension UIDevice {
+    class func formFactorString() -> String {
+        var formfactor: String
+        let FORMFACTOR_PHONE = "phone"
```

Comment:
Nit: Just for clarity around these constants, I would move them into an `enum` named something with some more context around what they are used along with a link like:

```swift
// Sync Object Format (Version 1) for Form Factors http://docs.services.mozilla.com/sync/objectformats.html#id2
enum SyncFormFactorFormat: String {
    case largeTablet = "largetablet"
    case smallTablet = "smalltablet"
    case phone = "phone"
}
```